webpack/envs/novoCommonConfig.js

```diff
@@ -0,0 +1,63 @@
+// @ts-check
+
+export const novoChunks = {
+  cacheGroups: {
+    vendors: false,
+    "arsty-common": {
+      chunks: "all",
+      enforce: true,
+      minChunks: 5,
+      minSize: 0,
+      name: "artsy-common",
+      reuseExistingChunk: true,
+      test: /.*src[\\/]/,
+    },
+    // "arsty-graphql-common": {
+    //   chunks: "all",
+    //   enforce: true,
+    //   priority: 50,
+    //   minChunks: 1,
+    //   minSize: 0,
+    //   name: "arsty-graphql-common",
+    //   test: /.*src[\\/]v2[\\/]__generated__[\\/]/,
+    // },
```

Comment:
Still relevant?

Reply:
Nah, it was useful to show that nearly 50% of our bundle source is duplicated GraphQL queries. I wouldn't use this in production.